src/app/app.component.html

```diff
@@ -1,3 +1,25 @@
 <ion-app>
-  <ion-router-outlet></ion-router-outlet>
+  <ion-menu side="start" menuId="m1">
+    <ion-header>
+      <ion-toolbar>
+        <ion-title>
+          Assembly Church
+        </ion-title>
+      </ion-toolbar>
+    </ion-header>
+    <ion-content>
+        <br/>
+        <ion-avatar margin>
+          <img src="https://cdn2.iconfinder.com/data/icons/avatar-profile/509/avatar_hair_contact_profile_user_default-512.png">
```

Comment:
Can you try using this `src` instead? 

https://via.placeholder.com/512

Reply:
i thought the vegeta avatar looked pretty cool  :(

Reply:
Nevermind! I was too scared to open it at work 😆 ignore me!